src/components/Select.vue

```diff
@@ -13,7 +13,7 @@
               :deselect="deselect"
               :multiple="multiple"
               :disabled="disabled">
-          <span class="vs__selected" v-bind:key="option.index">
```

Comment:
In this case `option.index` was not defined at all if I understood this correctly, since `option` can be anything (a string, object, number, ...) - whatever the developer passes. Please correct me if I am wrong.